testsuite/tests/libratest/smoke_test.rs

```diff
@@ -631,6 +634,11 @@ fn test_full_node_basic_flow() {
         Decimal::from_f64(30.0),
         Decimal::from_str(&validator_ac_client.get_balance(&["b", "4"]).unwrap()).ok()
     );
+
+    let sequence = full_node_client_2
```

Comment:
shouldn't you read actual sequence number from other node instead, where change is definitely already propagated ?

Reply:
true, makes sense to get from validator instead